tests/misc/BuildManPageTest.py

```diff
@@ -75,8 +75,8 @@ def test_mk_options(self):
     def test_mk_footer(self):
         uut = ManPageFormatter(app_name, ext_sections=sections)
         self.assertEqual(uut._mk_footer(),
-                         ".SH {0}\n"
-                         " {1}".format(section_name.upper(), section_text))
+                         ".SH {}\n {}".format(
```

Comment:
yeah but I thought the line break was not so good to read, and I directly removed the numbers inside {} as they are superfluous in this case :)
You think I should keep as it is?